diff --git a/root/game.py b/root/game.py
--- a/root/game.py
+++ b/root/game.py
@@ -813,7 +813,11 @@ class GameWindow(ui.ScriptWindow):
 	def __SidekickGear(self, slot="0", name="-", *rest):
 		import uisidekick
 		uisidekick.OnServerGear(slot, name)
 
+	def __SidekickGhost(self, vid="0", *rest):
+		import uisidekick
+		uisidekick.OnServerGhost(vid)
+
 	def __AutoHuntTarget(self, vid="0", *rest):
 		import uiautohunt
 		uiautohunt.OnServerTarget(vid)
@@ -2674,4 +2678,5 @@ class GameWindow(ui.ScriptWindow):
 			"SidekickGear"					: self.__SidekickGear,
 			"SidekickWindow"				: self.__SidekickWindow,
+			"SidekickGhost"					: self.__SidekickGhost,
 
 			# fishing
